nginx/conf/nginx.conf

```diff
@@ -0,0 +1,50 @@
+
+worker_processes  1;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+    sendfile        on;
+    keepalive_timeout  65;
+
+    server {
+        listen       8000 default_server;
+        server_name  localhost gtkm.com www.gtkm.com;
+        root         ..\gtkm\static;
+
+        location /api/auth/ {
+            proxy_pass http://localhost:8001;
+        }
+
+        location /api/stats/ {
+            proxy_pass http://localhost:8004;
+        }
+    }
+
+    server {
+        listen 8001;
+        listen 8002;
+        listen 8003;
+        listen 8004;
+        
```

Comment:
Wyjaśnisz mi tą część?

Czy to znaczy, że nginx ma słuchać też na portach 8001, 8002...?

Reply:
To jest do wyrzucenia